manim/utils/bezier.py

```diff
@@ -372,13 +372,13 @@ def is_closed(points: typing.Tuple[np.ndarray, np.ndarray]) -> bool:
     return np.allclose(points[0], points[-1])
 
 
-def bezier_params_from_point(
+def proportions_along_bezier_curve_for_point(
     point: typing.Iterable[typing.Union[float, int]],
     control_points: typing.Iterable[typing.Iterable[typing.Union[float, int]]],
     round_to: typing.Optional[typing.Union[float, int]] = 1e-6,
 ) -> typing.List[float]:
```

Comment:
```suggestion
) -> np.ndarray:
```

Reply:
Aargh. I keep forgetting stuff like this.